src/main/java/edu/kis/powp/jobs2d/command/visitor/Canvas.java

```diff
@@ -0,0 +1,18 @@
+package edu.kis.powp.jobs2d.command.visitor;
+
+
+import edu.kis.powp.jobs2d.drivers.transformation.Point;
+
+public class Canvas{
```

Comment:
This class represents a rectangular Canvas. I think Canvas should be an interface so other types of canvas could be implemented. Do you agree?

Reply:
added interface and renamed class RectangleCanvas